EventFilter/EcalRawToDigi/plugins/MatacqProducer.cc

```diff
@@ -18,7 +18,7 @@
 #include "DataFormats/EcalDigi/interface/EcalMatacqDigi.h"
 #include "DataFormats/FEDRawData/interface/FEDNumbering.h"
 #include "DataFormats/FEDRawData/interface/FEDRawData.h"
-#include "EventFilter/EcalRawToDigi/interface/MatacqProducer.h"
+#include "EventFilter/EcalRawToDigi/src/MatacqProducer.h"
```

Comment:
this is a header file for `class MatacqProducer : public edm::EDProducer {`
We usually move them to /plugins and include without a path.
Yet a better option is to merge .h in .cc and drop it from SealModules.cc

Reply:
done, I have merged the `.cc and .h` files for plugin and dropped `SealModules.cc`